contents/tutorials/ios-session-replay.md

```diff
@@ -0,0 +1,249 @@
+    }
+}
+```
+
+Next, create a new file `WelcomeViewController.swift` with the following:
+
+```swift file=WelcomeViewController.swift
+import UIKit
+
+class WelcomeViewController: UIViewController {
+    private let welcomeLabel = UILabel()
+    private let logoutButton = UIButton(type: .system)
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+    }
+
+    private func setupUI() {
+        view.backgroundColor = .white
+        title = "Welcome"
+
+        // Welcome Label
+        welcomeLabel.text = "Welcome!"
+        welcomeLabel.font = UIFont.systemFont(ofSize: 24, weight: .bold)
+        welcomeLabel.textAlignment = .center
+        view.addSubview(welcomeLabel)
+
+        // Logout Button
+        logoutButton.setTitle("Logout", for: .normal)
+        logoutButton.backgroundColor = .systemRed
+        logoutButton.setTitleColor(.white, for: .normal)
+        logoutButton.layer.cornerRadius = 10
+        logoutButton.addTarget(self, action: #selector(logoutButtonTapped), for: .touchUpInside)
+        view.addSubview(logoutButton)
+
+        setupConstraints()
+    }
+
+    private func setupConstraints() {
+        welcomeLabel.translatesAutoresizingMaskIntoConstraints = false
+        logoutButton.translatesAutoresizingMaskIntoConstraints = false
+
+        NSLayoutConstraint.activate([
+            welcomeLabel.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            welcomeLabel.centerYAnchor.constraint(equalTo: view.centerYAnchor),
+
+            logoutButton.topAnchor.constraint(equalTo: welcomeLabel.bottomAnchor, constant: 40),
+            logoutButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20),
+            logoutButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -20),
+            logoutButton.heightAnchor.constraint(equalToConstant: 44)
+        ])
+    }
+
+    @objc private func logoutButtonTapped() {
+        presentingViewController?.dismiss(animated: true, completion: nil)
+    }
+}
+```
+Our basic setup is now complete. Build and run your app to see it in action.
+
+![Video of basic iOS app setup](https://res.cloudinary.com/dmukukwp6/video/upload/v1725526241/posthog.com/contents/ios-sampleapp.mp4)
+
+## 2. Add PostHog to your app
+
+With our app set up, it’s time to install and set up PostHog. If you don't have a PostHog instance, you can [sign up for free](https://us.posthog.com/signup).
+
+First, add [`posthog-ios`](/docs/libraries/ios) as a dependency to your app using [Swift Package Manager](https://developer.apple.com/documentation/xcode/adding-package-dependencies-to-your-app) (or if you prefer, you can use [CocoaPods](/docs/libraries/ios#cocoapods)).
+
+To add the package dependency to your Xcode project, select `File > Add Package Dependency` and enter the URL `https://github.com/PostHog/posthog-ios.git`. Select `posthog-ios` and click Add Package.
+
+**Note:** Session replay requires SDK version `3.6.0` or higher.
```

Comment:
```suggestion
**Note:** Session replay requires SDK version `3.8.3` or higher.
```

lets use the latest since there were performance improvements and bug fixes already